google/cloud/pubsub_v1/publisher/_batch/base.py

```diff
@@ -109,32 +109,6 @@ def status(self):
         """
         raise NotImplementedError
 
-    def will_accept(self, message):
-        """Return True if the batch is able to accept the message.
-
-        In concurrent implementations, the attributes on the current batch
-        may be modified by other workers. With this in mind, the caller will
-        likely want to hold a lock that will make sure the state remains
-        the same after the "will accept?" question is answered.
-
-        Args:
-            message (~.pubsub_v1.types.PubsubMessage): The Pub/Sub message.
-
-        Returns:
-            bool: Whether this batch can accept the message.
-        """
-        # If this batch is not accepting messages generally, return False.
-        if self.status != BatchStatus.ACCEPTING_MESSAGES:
-            return False
```

Comment:
This statement moved into `publish()`, and the second one seems to be already there